### ISSUE TITLE
A hash sign in a property value results in incorrect .davmail.properties being saved

### ISSUE THREAD

**Issue by hategan** [mguessan/davmail#326]
The following line in .davmail.properties causes problems:
```
davmail.trayBackgroundColor=#c49fe2
```

Whenever `Settings.save()` is invoked, the value gets duplicated (e.g., `davmail.trayBackgroundColor=#c49fe2#c49fe2`). This appears to be caused by `Settings.convertLine()`, which considers hash signs in property values to introduce a comment. However, according to https://docs.oracle.com/cd/E23095_01/Platform.93/ATGProgGuide/html/s0204propertiesfileformat01.html, comments are only introduced by a hash sign in the beginning of a line.

**Comment by mguessan** [comment]
Correct, implementation was wrong and did not allow hash in comments.
Fix available in trunk

**Comment by mguessan** [comment]
Fix delvered, closing ticket
